src/abstractdataframe/io.jl

```diff
@@ -157,6 +157,10 @@ function html_escape(cell::AbstractString)
     cell = replace(cell, "&"=>"&amp;")
     cell = replace(cell, "<"=>"&lt;")
     cell = replace(cell, ">"=>"&gt;")
+    # Replace quotes so that the resulting string could also be used in the attributes of
+    # HTML tags
+    cell = replace(cell, "\""=>"&quot;")
+    cell = replace(cell, "'"=>"&apos;")
```

Comment:
I don't think adding additional escapes here should break anything for existing uses of `html_escape`. As a test, this `DataFrame` seems to `show` fine:

```
struct Foo{T} end
DataFrame(
    xs = ["'", "\"", "<foo>'</bar>"],
    ys = [Foo{'\''}(), Foo{'"'}, Foo{Symbol("\"'")}()],
    zs = Foo{'"'}[Foo{'"'}(), Foo{'"'}(), Foo{'"'}()],
)
```

Reply:
yes - I think it should be OK. Although it would be good to have a test if all is OK.

Reply:
I added this example in as an additional test case.

Of course, string comparison doesn't actually check if the resulting HTML makes sense (although in this case I've inspected and tested the output). As I was working through the tests, I did wonder whether, instead of comparing strings, it might be a good idea to run the outputs through a HTML parser and see (1) if it parses as valid HTML, and (2) check that the attributes and elements have the correct values.

Reply:
> although in this case I've inspected and tested the output

This is what I always do. I always feared that setting up the dependencies required to:

> run the outputs through a HTML parser and see (1) if it parses as valid HTML, and (2) check that the attributes and elements have the correct values.

would be too hard to maintain and too expensive (it is cheaper for me - as I am not an expert in the HTML parsing field - to just manually check it).

Agreed?